Dp-202-shop/server/src/db/models/OrderItem.model.ts

```diff
@@ -7,17 +7,15 @@ import {Order} from "./Order.model";
 export interface OrderItemAttributes {
   quantity: number;
   id?: number;
-  order_id?: number;
-  product_id?: number;
   createdAt?: Date;
   updatedAt?: Date;
```

Comment:
don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter